# This message is use to commmunicate betwwen the "Carte Balise" 
# and the Turtlebot

# We have 4 differents mode to control the LEDS

uint8 OFF		= 0
uint8 ALIVE		= 1
uint8 BLINK_SLOW	= 2
uint8 BLINK_FAST	= 3
uint8 R_ON		= 4
uint8 R_SLOW		= 5
uint8 R_FAST		= 6
uint8 ROSE		= 7

uint8 mode
